tests/ratchet/unreliable_msg_producer.c

```diff
@@ -183,13 +181,20 @@ produce_msg(unreliable_msg_producer_t *producer, vsc_buffer_t **plain_text, vscr
         node->msg->cipher_text = ratchet_message;
         node->msg->plain_text = plain_text_local;
 
-        produce_msg(producer, plain_text, msg);
+        produce_msg(producer, plain_text, msg, should_restore);
     } else {
         *msg = ratchet_message;
         *plain_text = plain_text_local;
 
+        TEST_ASSERT_EQUAL((vscr_ratchet_session_is_initiator(*producer->session) && !producer->sent_first_response)
+                                  ? vscr_msg_type_PREKEY
+                                  : vscr_msg_type_REGULAR,
+                vscr_ratchet_message_get_type(ratchet_message));
+
         producer->produced_count++;
 
         return;
     }
-}
\ No newline at end of file
+}
+
+#endif
```

Comment:
Code style: ensures that there is always at lease 1 empty line at the end of the file.